test/datastream_test.rb

```diff
@@ -5,33 +5,33 @@ class DatastreamTest < MiniTest::Test
   context 'scap content' do
     should 'be able to parse profiles' do
       parser = create_parser('ssg-rhel7-ds.xml')
-      profile_titles = [
-        "United States Government Configuration Baseline",
-        "Standard System Security Profile for Red Hat Enterprise Linux 7",
-        "Criminal Justice Information Services (CJIS) Security Policy",
-        "C2S for Red Hat Enterprise Linux 7",
-        "Health Insurance Portability and Accountability Act (HIPAA)",
-        "Unclassified Information in Non-federal Information Systems and Organizations (NIST 800-171)",
-        "DISA STIG for Red Hat Enterprise Linux 7",
-        "OSPP - Protection Profile for General Purpose Operating Systems v. 4.2",
-        "PCI-DSS v3 Control Baseline for Red Hat Enterprise Linux 7",
-        "Red Hat Corporate Profile for Certified Cloud Providers (RH CCP)",
-        "PCI-DSS v3 Control Baseline for Red Hat Enterprise Linux 7"
+      profile_ids = [
+        "xccdf_org.ssgproject.content_profile_ospp",
+        "xccdf_org.ssgproject.content_profile_standard",
+        "xccdf_org.ssgproject.content_profile_cjis",
+        "xccdf_org.ssgproject.content_profile_C2S",
+        "xccdf_org.ssgproject.content_profile_hipaa",
+        "xccdf_org.ssgproject.content_profile_nist-800-171-cui",
+        "xccdf_org.ssgproject.content_profile_stig-rhel7-disa",
+        "xccdf_org.ssgproject.content_profile_ospp42",
+        "xccdf_org.ssgproject.content_profile_pci-dss",
+        "xccdf_org.ssgproject.content_profile_rht-ccp"
       ]
-      assert_equal(profile_titles, parser.profiles.map(&:title))
+      assert_equal(profile_ids, parser.benchmark.profiles.map(&:id))
     end
   end
 
-  context 'tailoring file' do
```

Comment:
@xprazak2 I'm not sure who uses openscap_parser for tailoring, but it's a whole different file type from Datastream. The profiles are not `<Profile...` but `<profile ...` as they aren't defining a profile but selecting/deselecting Rules from that Profile. It wouldn't be very difficult to patch up the tailoring workflow - but I'm just wondering if it's used anywhere you know.

Reply:
I think nobody uses it at the moment, but [smart_proxy_openscap](https://github.com/theforeman/smart_proxy_openscap/pull/71/files#diff-d9b04fc460aeec56309cdf7a9eb41a33R139) plans to :grin:.

But I agree it is a different file type and we should find a better way how to handle it rather than lump it together with Datastream.

Reply:
We'll be starting to implement tailoring later this month, so I'll revisit this then.